osdconfig/api.go

```diff
@@ -310,15 +357,23 @@ func (manager *configManager) run(wd *DataWrite) {
 			if val.Type == wd.Type {
 				select {
 				case <-manager.runCtx.Done():
-					logrus.Info("context cancellation received in: callback scheduler")
-					logrus.Error("not all callbacks delivered completion status")
+					logrus.WithField("source", sourceManagerRun).
+						Warn(msgCtxCancelled)
+					logrus.WithField("source", sourceManagerRun).
+						Warn(msgStatusError)
 					break Loop
 				case mesg := <-readFanIn:
 					name, err, dur := mesg.Name, mesg.Err, time.Since(t)
 					if err != nil {
-						logrus.Error(mesg.Name, " callback completed w/  errors: ", dur)
+						logrus.WithField("source", sourceManagerRun).
```

Comment:
Errors are warnings now throughout the code?

Reply:
yes, this allows us to focus on scheduler errors by utilizing logrus log levels. Not all errros are showstoppers.